std/meta.d

```diff
@@ -1309,6 +1309,52 @@ private template staticMerge(alias cmp, int half, Seq...)
     }
 }
 
+/**
+ * Allows subsetting of multiple members of tuple types that are not
```

Comment:
Per dlang.org/tuple, don't use the term "tuple" for this. I think "alias sequence" is a good fit here.